tests/manage/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -97,20 +98,22 @@ def test_non_ocs_taint_and_tolerations(self):
         )
         storagecluster_obj.patch(params=param, format_type="merge")
 
-        # Add tolerations to the subscription
-        sub_list = ocp.get_all_resource_names_of_a_kind(kind=constants.SUBSCRIPTION)
-        param = (
-            '{"spec": {"config":  {"tolerations": '
-            '[{"effect": "NoSchedule", "key": "xyz", "operator": "Equal", '
-            '"value": "true"}]}}}'
+        # Add tolerations to the configmap rook-ceph-operator-config
+        configmap_obj = ocp.OCP(
+            kind=constants.CONFIGMAP,
+            namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            resource_name=constants.ROOK_OPERATOR_CONFIGMAP,
         )
-        for sub in sub_list:
-            sub_obj = ocp.OCP(
-                resource_name=sub,
-                namespace=defaults.ROOK_CLUSTER_NAMESPACE,
-                kind=constants.SUBSCRIPTION,
-            )
-            sub_obj.patch(params=param, format_type="merge")
+        toleration = configmap_obj.get().get("data").get("CSI_PLUGIN_TOLERATIONS")
+        toleration += (
+            '\n- key: xyz\n  operator: Equal\n  value: "true"\n  effect: NoSchedule'
+        )
+        toleration = toleration.replace('"', '\\"').replace("\n", "\\n")
+        param_cmd = (
+            f'[{{"op": "replace", "path": "/data/CSI_PLUGIN_TOLERATIONS", "value": "{toleration}" }}, '
+            f'{{"op": "replace", "path": "/data/CSI_PROVISIONER_TOLERATIONS", "value": "{toleration}" }}]'
+        )
+        configmap_obj.patch(params=param_cmd, format_type="json")
 
```

Comment:
Please add logging after every resource successful patch operation .
log.info(f"Successfully added toleration to {sub}")
Such logging info can help while analysing the test failure.

Reply:
Done